docs/source/posts/2024/2024-06-26-week3-wachiou-bouraima.rst

```diff
@@ -0,0 +1,74 @@
+WEEK 3: Refinements and Further Enhancements
```

Comment:
![image](https://github.com/fury-gl/fury/assets/115863770/b4a16ef7-438b-4438-ac5a-dc07c12f7a50)

`Title underline too short` for `Hello everyone,`. Can be fixed by adding an extra `-`.